Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -51,19 +75,17 @@ Creating a new Symfony Command in Your Extension
 
    :php:`configure()`
       As the name would suggest allows to configure the command.
-      Allows to add a description or a help text, or mandatory and optional
-      arguments and parameters defined.
+      Allows to add a description or a help text,
+      or mandatory and optional arguments and parameters defined.
```

Comment:
minor nitpick (this was already this way before): how can arguments be mandatory _and_ optional? I would just simplify it (the point that you can have mandatory or optional arguments will reveal itself in the code snippet below).

> Allows to add a description, a help text or define arguments for the command.